packages/rest/src/writer.ts

```diff
@@ -59,17 +60,16 @@ export function writeResultToResponse(
  */
 export function writeErrorToResponse(response: Response, error: Error) {
   const e = <HttpError>error;
-  const statusCode = (response.statusCode = e.statusCode || e.status || 500);
   if (e.headers) {
```

Comment:
By moving to strong-error-handler, we will loose this feature. However, I believe were not using `error.headers` field so far (LB 3.x, 4.x), thus it should not be a problem. We can always add support for `e.headers` later in strong-error-handler if needed.